### PR TITLE
[CPU] add const and precision check quantize ranges

### PR DESCRIPTION
**Pull request by mandrono**

### CODE REVIEW

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.cpp [hunk 1]
```diff
@@ -124,6 +124,14 @@ void MKLDNNQuantizeNode::init() {
             THROW_IE_EXCEPTION << "Unsupported input sizes for Quantize layer with name " << getName();
     }
 
+    for (size_t i = 1; i < getParentEdges().size(); i++) {
+        if (!getParentEdgesAtPort(i)[0]->getParent()->isConstant())
+            THROW_IE_EXCEPTION << "Quantize layer with name " << getName() << " has non const input on " << i << " port";
+        auto prec = getCnnLayer()->insData[i].lock()->getPrecision();
+        if (prec != Precision::FP32)
+            THROW_IE_EXCEPTION << "Quantize layer with name " << getName() << " has unsupported precision " << prec << " on " << i << " port";
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.cpp:132]: Lets write which exact precision it is.

Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.cpp:132]: it indicated after ```has unsupported precision```
```prec``` print

Comment (dmitry-gorokhov) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.cpp:132]: Ok. My bad

**Review by dmitry-gorokhov — APPROVED** [review]

**Comment by dmitry-gorokhov** [comment]
@mandrono  Please rebase your PR on latest master

**Pull request merged**